presto-pinot-toolkit/src/main/java/com/facebook/presto/pinot/PinotConfig.java

```diff
@@ -442,4 +443,16 @@ public PinotConfig setInferTimestampTypeInSchema(boolean inferTimestampTypeInSch
         this.inferTimestampTypeInSchema = inferTimestampTypeInSchema;
         return this;
     }
+
+    public boolean isRetryPinotException()
```

Comment:
RetryPinotException conveys to me that the pinot exception would be retried.

How about the name: isMarkDataFetchExceptionsAsRetriable ?